documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -73,10 +73,11 @@ Polaris now supports multiple languages and ships with [many translations](https
 
 // new
 import translations from '@shopify/polaris/locales/en.json';
-
 <AppProvider i18n={translations}>
 ```
 
+If you wish to dynamically load translations based on a provided locale see the [AppProvider README](https://github.com/Shopify/polaris-react/blob/master/src/components/AppProvider/README.md#using-translations) for more information.
```

Comment:
Should we mention it's only relevant if you use `@shopify/react-i18n`?

Reply:
We mention that in the README, people will find out soon enough.

Reply:
It seems odd to me we're directing people to another page, then letting them know we actually meant we can show you how to use dynamically load translations if you use our internationalization package and our non-opened sourced tool without public docs or webpack (with maybe custom config required). Keeping in mind the majority of projects consuming polaris are not internal.

Reply:
Sadie brought this up as well. let's add a note in :)